package.json

```diff
@@ -10,47 +10,49 @@
     "packages/*"
   ],
   "scripts": {
-    "clean": "rm -rf node_modules server/node_modules client/node_modules packages/node_modules package-lock.json yarn.lock server/package-lock.json client/package-lock.json packages/package-lock.json server/yarn.lock client/yarn.lock packages/yarn.lock server/dist client/dist packages/dist",
-    "build:prod": "yarn && yarn build:tsc && yarn build:expo",
+    "clean:all": "npm run clean:packages && npm run clean:client && npm run clean:server && rm -rf package-lock.json npm run.lock node_modules",
+    "clean:packages": "rm -rf packages/validations/node_modules packages/validations/package-lock.json packages/validations/dist",
+    "clean:client": "rm -rf client/node_modules client/package-lock.json client/dist",
+    "clean:server": "rm -rf server/node_modules server/package-lock.json server/dist",
+    "build:prod": "npm run && npm run build:tsc && npm run build:expo",
     "build:prod:docker": "docker build -t packrat-app -f Dockerfile.prod .",
-    "build:expo": "yarn expo:export",
-    "build:tsc": "yarn build:tsc:server",
-    "build:tsc:packages": "cd packages/validations && yarn build",
-    "build:tsc:server": "cd server && yarn build",
-    "install:all": "yarn install:packages && yarn install:client && yarn install:server",
-    "install:client": "cd client && yarn",
-    "install:server": "cd server && yarn",
-    "install:packages": "cd packages && yarn",
-    "setup": "yarn && yarn setup:client && yarn setup:server",
-    "setup:client": "cd client && yarn setup",
-    "setup:server": "cd server && yarn setup",
+    "build:expo": "npm run expo:export",
```

Comment:
i'm not sure i want to switch our package manager to npm again, yarn seems to definitely have some benefits

Reply:
Yarn could did not install dependencies for me in client directory nor server direction.
It cached install state and had all dependencies in parent directory yet ios build could not find the dependencies in the client directory resulting in ios build failure.

Reply:
Let me try it again let you know.

Reply:
From the build log I saw hermesEnabled is required by some libraries. Please include it in gradle.properties file..